README.md

```diff
@@ -39,7 +39,54 @@ proxy is required that runs in front of Kibana.
 The deployer enables the user to generate all of the necessary
 key/certs/secrets and deploy all of the components in concert.
 
-## Defining local builds
+#### Curator
```

Comment:
correct, we evaluate project names with ".*" appended to account for the date format at the end, and the inability for project names to contain a period (when created through OpenShift)